src/assets/styles.scss

```diff
@@ -18,6 +18,11 @@ $addColors: (
 // Merge default colors with our custom choices
```

Comment:
Not sure why there is a usage of `!important` here, it seems sorta dangerous.

Reply:
you might've been right about that! i used `!important` because it allowed me to override base css stylings directly without making my css selectors super specific. i removed all the `!important` stylings and made them all more specific, so hopefully more styles don't need to be added in the future.